docs/release-notes.asciidoc

```diff
@@ -7,6 +7,55 @@
+* Adds pre-packaged rule updates through the "Prebuilt Security Detection Rules" Fleet integration ({pull}96698[#96698]).
+* Filters the Alerts table by threat presence of threat ({pull}96096[#96096]).
+* Populates `threat.indicator.event` with `source.event` data ({pull}95697[#95697]).
+* Adds the threat summary to the *Summary* tab in the Alert details flyout and introduces the *Threat Intel* tab ({pull}95604[#95604]) ({pull}97185[#97185]).
+* Updates Cloud plugin to handle new config values in kibana.yml ({pull}95569[#95569]).
+
+[discrete]
+[[bug-fixes-7.13.0]]
+==== Bug fixes and enhancements
+* Fetches detection adoption metrics  ({pull}97789[#97789]).
+* Updates fields with Beats metadata ({pull}97719[#97719]).
+* Updates detection alert mappings to ECS 1.9 ({pull}97573[#97573]).
+* ML rules accept multiple ML job IDs ({pull}97073[#97073]).
+* Adds the Security Network ML Module to the list of available jobs ({pull}97014[#97014]).
+* Updates MITRE tactics, techniques, and subtechniques ({pull}97011[#97011]).
+* Improves user experience duplicating rules ({pull}96760[#96760]).
+* Introduces a nested CTI row renderer ({pull}96275[#96275]).
+* Rebuilds nested fields structure from field's response ({pull}96187[#96187]).
+* Combines multiple timestamp searches into a single request ({pull}96078[#96078]).
+* Adds the Indicator Match Timeline template ({pull}95840[#95840]).
+* Fetches additional detection rule adoption metrics ({pull}95659[#95659]).
+* Adds HTTP endpoints for the Timeline ({pull}95036[#95036]).
+* Updates the agent status labels and colors ({pull}99314[#99314]).
+* Fixes an issue where many `OR` clauses take up too much vertical space ({pull}98706[#98706]).
+* Adds network responses to error toasters ({pull}97945[#97945]).
+* Fixes issue where long hostnames were truncated in the agent detail flyout.({pull}97253[#97253]).
+* Fixes a bug with DNS query that caused additional terms to be accidentally requested. ({pull}97069[#97069]).
+* Allows a preview of query results when creating a new rule or editing an existing one. ({pull}94018[#94018]).
+* Fixes the rule details page to show the rule details loading when the *Activated* switch is toggled. ({pull}94010[#94010]).
+* Sets the default date time on the timepicker to `today` instead of `Last 24 hours` to enable cachability. Also fixes a date math bug in the URL ({pull}93548[#93548]).
+* Fixes size issue with detection rule telemetry ({pull}99900[#99900]).
+* Excludes meta fields from the fields API request({pull}99443[#99443]).
+
+[discrete]
+[[known-issues-7.13.0]]
+==== Known issues
+* A histogram cannot be generated for these fields because their mappings have changed ({pull}670[#670]): 
```

Comment:
The wording looks great 👍 

The link for `670` takes me to an issue in the kibana repo that doesn't seem correct

https://github.com/elastic/kibana/issues/670

Reply:
Verified was ok to remove link from the release notes summary. Thanks for the review @jonathan-buttner!